### PR TITLE
fix(waku): link props

### PR DESCRIPTION
**Pull request by himself65**
@sandren

### CODE REVIEW

File: packages/waku/src/router/client.ts [hunk 1]
```diff
@@ -11,7 +11,15 @@ import {
...
   Fragment,
 } from 'react';
-import type { ComponentProps, FunctionComponent, ReactNode } from 'react';
+import type {
+  ComponentProps,
+  FunctionComponent,
+  ReactNode,
+  AnchorHTMLAttributes,
+  ReactElement,
+  PropsWithChildren,
```
Comment (dai-shi) [inline packages/waku/src/router/client.ts:20]: I slightly prefer `children: ReactNode` for explicitness.

File: packages/waku/src/router/client.ts [hunk 2]
```diff
@@ -69,19 +77,23 @@ export function useLocation() {
   return value.loc;
 }
 
+export type LinkProps = PropsWithChildren<
+  {
+    href: string;
```
Comment (dai-shi) [inline packages/waku/src/router/client.ts:82]: Should we change this to `to`? v0.19.0 would be a good timing for such a breaking change.

**Review by dai-shi — COMMENTED** [review]
`packages/waku/src/router/client.ts` looks good to me.

**Comment by himself65** [comment]
hey we need rerun the error CI

**Review by sandren — APPROVED** [review]
> @sandren

Thanks! 🙂

Yes, let's use `to` for the Waku Router API since it's more or less the React community standard (React Router, Remix, Gatsby, TanStack Router, etc).

**Pull request merged**